app/assets/stylesheets/application.css.scss

```diff
@@ -11,6 +11,7 @@
  * It is generally better to create a new file per style scope.
  *
  *= require font-awesome
+ *= require variables
```

Comment:
Thanks! I actually just got some help with this from a colleague (who has been trying to get me to use sass for ages).